test/factories/users.rb

```diff
@@ -23,6 +24,8 @@
       |n| "nickname#{n}"
     end
     club_id 1
+    admin false
+    belonging_to { ('a'..'z').to_a.sample(15).join }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.